internal/spec/filebeat.yml

```diff
@@ -5,7 +5,7 @@ args: [
   "-E", "setup.template.enabled=false",
   "-E", "management.enabled=true",
   "-E", "logging.level=debug",
-  "-E", "gc_percent=${FILEBEAT_GOGC:100}"
+  "-E", "gc_percent=${FILEBEAT_GOGC:100}",
```

Comment:
o_O, that should have raised a check issue elsewhere.

Reply:
@narph This missing ',' is strange to me and also the line disabling the `modules` they are not present in the original file https://github.com/elastic/beats/blob/main/x-pack/elastic-agent/spec/filebeat.yml#L6-L9 ?